src/control-plane/backend/lambda/api/router/segment.ts

```diff
@@ -33,7 +33,7 @@ const segmentServ = new SegmentServ();
 router_segment.post(
   '',
   validate([
-    body().custom(isValidEmpty).custom(isXSSRequest),
```

Comment:
why removing this check?

Reply:
I found the segment request failed sometimes due to this check even the segment is pretty simple. So I remove it as it's unlikely to have XSS attack in create segment api request

Reply:
I did more tests, but couldn't reproduce the above issue. Add the XSS check back.